Antidote/ActiveCallViewController.m

```diff
@@ -197,6 +202,27 @@ - (void)createBadgeViews
     NSInteger numberOfPausedCalls = [self.dataSource activeCallControllerNumberOfPausedCalls:self];
     self.badgeLabel.text = [NSString stringWithFormat:@"%lu", numberOfPausedCalls];
 }
+
+- (void)createFriendAvatar
+{
+    AvatarsManager *avatars = [AppContext sharedContext].avatars;
+
+    UIImage *image = [avatars avatarFromString:self.nickname
+                                      diameter:kAvatarDiameter
+                                     textColor:[UIColor whiteColor]
+                               backgroundColor:[UIColor clearColor]];
+
+    self.friendAvatar = [[UIImageView alloc] initWithImage:image];
+
+    [self.view addSubview:self.friendAvatar];
+
+    [self.friendAvatar makeConstraints:^(MASConstraintMaker *make) {
+        make.centerX.equalTo(self.view);
+        make.centerY.equalTo(self.view);
```

Comment:
You can just use `center` instead of `centerX` and `centerY`. However it isn't critical.